UI/org.eclipse.birt.report.debug.core/plugin.xml

```diff
@@ -1,5 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.2"?>
+<!--
+ *******************************************************************************
+ * Copyright (c) 2021 Contributors to Eclipse Foundation
```

Comment:
Steve, should this not be "the" Eclipse Foundation? I'm not a native speaker but this looks like a spelling error. What do you think?

Reply:
You're right.  I'll change the scripts.

Reply:
As I exercise the scripts on more projects, should I continue with this PR or create a new ones?

Reply:
I don't understand. Let's focus on this project first. If all is fine then you can make separate pull requests for other projects.

Please also note the build.properties file. It should contain entries for the new license files and entries for removed files should be removed.

Reply:
@wimjongman When I say project, I'm referring to eclipse projects within the BIRT repo.  There are 297 of them.  @ruspl-afed suggested we do one PR per bundle, which is roughly equivalent to an eclipse project.  We are done with this bundle, correct?

Reply:
We are not done yet. There is still this one:


Steve, should this not be "the" Eclipse Foundation? I'm not a native speaker but this looks like a spelling error. What do you think?
Member Author
@SteveSchafer-Innovent SteveSchafer-Innovent 3 days ago

You're right. I'll change the scripts

Reply:
I needed to correct them manually but the scripts will pick them up going forward.

Reply:
Just noticed another wrinkle.  There are 29 instance of "about.properties" that contain license text in a property value.  The scripts don't detect this because they are looking for comments.  I should fix that before we merge this PR.

Reply:
I notice that the file "about.properties" contains a copyright notice in a property value, but my script added a copyright notice as a comment to the file itself because it didn't have one, and these two notices don't match exactly.  I'm not sure if that's a problem since the comment I added is a copyright for the about.properties file and the copyright notice in the property value is a copyright for the eclipse project as a whole.